README.md

```diff
@@ -145,14 +198,19 @@ id                 Convert jobid(s) to another form
 overlay            Show flux overlay network status
 kvs                Flux key-value store utility
 pstree             display job hierarchies
+resource           List/manipulate Flux resource status
```

Comment:
Maybe we want to keep this? It's not a system information thing, ti's a resource _controller_ thing, e.g., look at the other subcommands:

```
$ flux resource --help
usage: flux-resource [-h] {drain,undrain,status,list,info,reload} ...

optional arguments:
  -h, --help            show this help message and exit

subcommands:

  {drain,undrain,status,list,info,reload}
```

Reply:
Diffs make this a little ambiguous, but I just moved this command to the system hierarchy since it felt better grouped with `start`, `stop`, and the other commands that modify the flux instances themselves. Do you think it should instead be in the root hierarchy?

Reply:
So you are saying we would do `flux system list` `flux system reload` etc?

Reply:
ping @alecbcs !

Reply:
Ah! No I'm saying we should put resource under the system hierarchy we've already made so the command would be,
```console
$ flux system resource list/reload/etc...
``` 

just like,

```console
$ flux system start/stop
```

Reply:
Why do we need system and resource? That's a lot of subcommands. I would vote for either leaving system where it is (which I think the flux devs would like) it flattening it under system.

Reply:
Based on the mattermost discussion I've reverted resource to the top level. `id` also no longer felt like a system command so I've moved that back up to the top level as well.

Reply:
great, thank you!